lib/common/widgets/game_finish_dialog.dart

```diff
@@ -14,110 +14,114 @@ Future<void> showGameFinishDialog({
     barrierDismissible: false,
     builder: (context) => Dialog(
       backgroundColor: Colors.transparent,
-      child: Builder(builder: (context) {
-        final sortedItems = items.sortedByType();
+      child: Builder(
+        builder: (context) {
+          final sortedItems = items.sortedByType();
 
-        return ConstrainedBox(
-          constraints: const BoxConstraints(
-            maxWidth: 600,
-            maxHeight: 600,
-          ),
-          child: Container(
-            padding: const EdgeInsets.all(24),
-            decoration: BoxDecoration(
-              borderRadius: BorderRadius.circular(40),
-              border: Border.all(
-                width: 2,
-                color: FlutterGameChallengeColors.textStroke,
-              ),
-              color: FlutterGameChallengeColors.detailsBackground,
+          return ConstrainedBox(
+            constraints: const BoxConstraints(
+              maxWidth: 600,
+              maxHeight: 600,
             ),
-            child: Column(
-              mainAxisSize: MainAxisSize.min,
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                const SizedBox(height: 12),
-                Flexible(
-                  child: Text(
-                    context.l10n.gameFinishDialogTitle,
-                    style: const TextStyle(
-                      color: FlutterGameChallengeColors.textStroke,
-                      fontSize: 36,
+            child: Container(
+              padding: const EdgeInsets.all(24),
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(40),
+                border: Border.all(
+                  width: 2,
+                  color: FlutterGameChallengeColors.textStroke,
+                ),
+                color: FlutterGameChallengeColors.detailsBackground,
+              ),
+              child: Column(
+                mainAxisSize: MainAxisSize.min,
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  const SizedBox(height: 12),
```

Comment:
You have the same vertical padding at like 52, consider moving this value to a const.

Reply:
added spacing to the constants [Added const value for dialog content inner padding](https://github.com/DenisovAV/recyclo/pull/19/commits/9a6550a052b963961c7107ab3b167859b76fa0de)